.eclipse-pass.JHUAWSdemo_env

```diff
@@ -1,10 +1,13 @@
 ###################################################
 # PASS_CORE config ################################
 ###################################################
-PASS_CORE_BASE_URL=https://demo.eclipse-pass.org
+
+PASS_CORE_BASE_URL=https://demo.eclipse-pass.org:8080
```

Comment:
We will have to take a note to update the Staging and Production env files in the JHU repo too before deployment happens.